framework/h/xhr.go

```diff
@@ -34,7 +35,7 @@ func PostPartialWithQs(partial PartialFunc, qs *Qs, trigger ...string) *Attribut
 	if !strings.HasPrefix(path, "/") {
 		path = "/" + path
 	}
-	return Post(GetPartialPathWithQs(partial, qs), trigger...)
```

Comment:
actually would you mind removing the HasPrefix check? GetPartialPathWithQs adds the slash now. I added that after the fact and forgot to remove this code

Reply:
ok